### PR TITLE
correct problems in orthoprofile and code

### PR DESCRIPTION
**Pull request by LinguList**
see #14

### CODE REVIEW

**Comment by LinguList** [comment]
Should I also update the CLDF now?

**Comment by XachaB** [comment]
The changes to the profile are important (due to re-ordering the rows and/or removing some columns, I think), making it difficult to review it more, but I find no more intervocalic [y] after this, so to me the changes look good.

File: TRANSCRIPTION.md [hunk 1]
```diff
@@ -329,10 +329,10 @@
 | aͥ/a | 2 | ✓ | ✓ |
 | bʲ | 2 | ✓ | ✓ |
 | f̩ | 2 | ✓ | ✓ |
-| jː | 2 | ✓ | ✓ |
 | o̯/o | 2 | ✓ | ✓ |
 | rː | 2 | ✓ | ✓ |
 | wʰ | 2 | ✓ | ✓ |
+| yː | 2 | ✓ | ✓ |
```
Comment (XachaB) [inline TRANSCRIPTION.md:335]: Are you sure that this was indeed [y] and not [j], right ?  

I find these:

```
tauade-four-1,218180,tauade,four,'aiy:ɔ,'aiy:ɔ,a i yː ɔ,,sil-1980,,,^ ' a i y: ɔ $,default
tauade-ten-1,218182,tauade,ten,'aiy:ɔ kupařima,'aiy:ɔ_kupařima,a i yː ɔ + k u p a r̝ i m a,all of us,sil-1980,,,^ ' a i y: ɔ _ k u p a ř i m a $,default
```

File: TRANSCRIPTION.md [hunk 2]
```diff
@@ -373,12 +373,12 @@
...
 | sˤ | 1 | ✓ | ✓ |
 | s̩ | 1 | ✓ | ✓ |
-| yˤ | 1 | ✓ | ✓ |
+| y̶ᶻ/jˤ | 1 | ✓ | ✓ |
 | à:/aː | 1 | ✓ | ✓ |
 | ãu | 1 | ✓ | ✓ |
 | í:/iː | 1 | ✓ | ✓ |
 | ù̠/u | 1 | ✓ | ✓ |
-| ÿ/j | 1 | ✓ | ✓ |
+| ÿ/y | 1 | ✓ | ✓ |
```
Comment (XachaB) [inline TRANSCRIPTION.md:381]: Is this also correct ? the single occurence is in:

```
wipi-zim-arm-1,227237,wipi-zim,arm,ÿim,ÿim,ÿ/y i m,,fleischmann-and-turpeinen-1976,,,^ ÿ i m $,default
```

**Comment by LinguList** [comment]
Yes, it is then just some weird double dots, but typicallly, in SA 
langauges, ÿ refers to y.

**Comment by XachaB** [comment]
Ok, if you checked these two cases and are happy with them, can you re-request the review, so I can approve ?

**Review by XachaB — APPROVED** [review]

**Comment by XachaB** [comment]
Ok, I found how to give an approving re-review, is this enough for you to merge?

**Pull request merged**